scio-bigquery/src/main/scala/com/spotify/scio/bigquery/BigQueryClient.scala

```diff
@@ -197,8 +197,29 @@ class BigQueryClient private (private val projectId: String,
       getTable(table).getSchema
     }
 
-  /** Execute a query and save results into a temporary table. */
-  def queryIntoTable(sqlQuery: String): QueryJob = {
+  /** Make a query and save results to a destination table. */
+  def queryIntoTable(sqlQuery: String,
+                     destinationTable: TableReference,
+                     flattenResults: Boolean): Job = {
```

Comment:
`queryIntoTable` is overloaded and 2 versions cannot have the same number of arguments with optional `flattenResults`, or it'll cause ambiguous overloading error.